app/views/events/show.html.erb

```diff
@@ -0,0 +1,26 @@
+   <div class="row">
+     <div class="col-md-6 ">
+       <h2>Attendees for event:</h2>
+       <h4>" <%= @event.name %> "</h4><br>
+       <ul>
+          <% @event.users.each do |users| %>
+          <li><b><p><%= users.name %></p></b></li>
+           <% end %>
+       </ul>
+      </div>
+      <% if current_user.id == @event.creator_id %>
+       <div class="col-md-6 ">
+         <h2>Invite Users</h2>
+         <ul id="invite-users" >
+          <% @users.each do |user| %>
+            <% if !@event.users.exists?(id: user.id) %>
```

Comment:
Same here.
1. Retrieving **ALL** the users that exist on the application and checking one by one if it's an attendee seems wasteful. Associations should solve this problem in a better way.
2. If you were to do this comparison I'd rather have it in a place that's not the view